docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md

```diff
@@ -0,0 +1,335 @@
+---
+layout: docs
+title: Helper Methods
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/helper-methods
+---
+# Helper Methods
+
+The helper methods are used while [implementing a reporter](creating-reporter-plugin.md#implementing-reporter)
+to format a report output. TestCafe mixins these methods to the reporter.
+
+To refer to the helper methods, use `this`.
```

Comment:
A have always thought that it goes this way: we use `this` to *refer to* the current object; and we also use it to *access* the current object's fields, including helper methods.
Prove me I'm wrong pls)